### DISCUSSION TITLE
Inquiry about Feed Forward and Backward Processes in Chronos Model

### DISCUSSION THREAD

**Discussion by yogipuji15** [amazon-science/chronos-forecasting#180] [Q&A]
Hello,

I hope this message finds you well. I am currently exploring the Chronos model and I would like to gain a deeper understanding of the feed forward and backward processes within the model. Specifically, could you provide insights on how the feed forward and backward passes are handled in the implementation?

Are there any specific layers or methods in the model where these processes can be observed or logged?

Thank you for your assistance!

**Comment by abdulfatir** [comment]
Chronos models available in this repo are based on the T5 architecture and use huggingface transformers under the hood. You can check the implementation of T5 in transformers here: https://github.com/huggingface/transformers/blob/main/src/transformers/models/t5/modeling_t5.py
